### ISSUE TITLE
Rxjs is not included in deps

### ISSUE THREAD

**Issue by patrickml** [glebbash/nestjs-swagger-dto#20]
I believe I am getting this error because of using a monorepo and being forced to use nohoisting on nest js. Because of this each modules dependancies are track differently and cannot rely on the chance another module imports something it depends on.

This package has "rxjs": "^7.8.1", as a dev dependancy but it uses it directly within the package code within the IsNested decorator to perform a noop https://github.com/glebbash/nestjs-swagger-dto/blob/master/src/decorators/is-nested.ts#L3

```
Error: Cannot find module 'rxjs'
Require stack:
- /Users/patricklewis/projects/birdbrains/node_modules/.store/nestjs-swagger-dto-virtual-82554d1806/package/lib/decorators/is-nested.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1248:15)
    at Module._load (node:internal/modules/cjs/loader:1074:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Module.require (node:internal/modules/cjs/loader:1339:12)
    at require (node:internal/modules/helpers:125:16)
    at Object.<anonymous> (/Users/patricklewis/projects/birdbrains/node_modules/.store/nestjs-swagger-dto-virtual-82554d1806/package/src/decorators/is-nested.ts:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/patricklewis/projects/birdbrains/node_modules/.store/nestjs-swagger-dto-virtual-82554d1806/package/lib/decorators/is-nested.js'
  ]
}
```

**Comment by glebbash** [comment]
Did you try explicitly adding rxjs to package.json as a workaround?

But I'll get rid of the noop import soon anyways, thanks for reporting.

**Comment by glebbash** [comment]
Fixed in 3.8.2, sorry it took so long
